examples/fenda_example/client.py

```diff
@@ -26,7 +26,7 @@ def __init__(
         device: torch.device,
         minority_numbers: Set[int],
     ) -> None:
-        super().__init__(data_path=data_path, metrics=metrics, device=device)
+        super().__init__(data_path=data_path, metrics=metrics, device=device, perfcl_loss_weights=(1.0, 1.0))
```

Comment:
Adding in the perfcl loss to the FENDA example. This is just for testing when running the example to make sure nothing is broken there.